Sources/AlertToast/AlertToast.swift

```diff
@@ -682,6 +682,35 @@ public extension View{
         modifier(AlertToastModifier(isPresenting: isPresenting, duration: duration, tapToDismiss: tapToDismiss, offsetY: offsetY, alert: alert, onTap: onTap, completion: completion))
     }
     
+    /// Present `AlertToast`.
+    /// - Parameters:
+    ///   - item: Binding<Item?>
+    ///   - alert: (Item?) -> AlertToast
+    /// - Returns: `AlertToast`
+    func toast<Item>(item: Binding<Item?>, duration: Double = 2, tapToDismiss: Bool = true, offsetY: CGFloat = 0, alert: @escaping (Item?) -> AlertToast, onTap: (() -> ())? = nil, completion: (() -> ())? = nil) -> some View{
```

Comment:
You might want to make `Item` identifiable like in SwiftUI's alert?
```suggestion
    func toast<Item>(item: Binding<Item?>, duration: Double = 2, tapToDismiss: Bool = true, offsetY: CGFloat = 0, alert: @escaping (Item?) -> AlertToast, onTap: (() -> ())? = nil, completion: (() -> ())? = nil) -> some View where Item : Identifiable {
```